### PR TITLE
Add ffmd-packages repository to site stream

### PR DESCRIPTION
**Pull request by penguineer**
add ffmd modules according to https://gluon.readthedocs.io/en/v2016.2.x/user/site.html#site-modules

### CODE REVIEW

**Comment by penguineer** [comment]
A build using this changes was successful. There shouldn't be any effective changes to the firmware.

File: modules [hunk 1]
```diff
@@ -0,0 +1,5 @@
+GLUON_SITE_FEEDS='ffmd'
+
+PACKAGES_FFMD_REPO=https://github.com/FreifunkMD/ffmd-packages
```
Comment (LeSpocky) [inline modules]: This should be a valid Git repository URL? What you copy from GitHub to clone is https://github.com/FreifunkMD/ffmd-packages.git

Comment (penguineer) [reply modules]: They both work and I definitely test-built it (used for package development).

Github directly resolves the *.git-URLs to the format I gave. What is the rationale in adding .git?

**Comment by LeSpocky** [comment]
> Github directly resolves the *.git-URLs to the format I gave. What is the rationale in adding .git?

I just thought it is easier recognizable as URL of a Git repository.
As stated by myself, both work. Use whatever you like. :-)

**Comment by penguineer** [comment]
Successfully built and tested with this addition.

**Pull request merged**